### DISCUSSION TITLE
Any way for this library to work without the active learning part i.e. in an unsupervised manner?

### DISCUSSION THREAD

**Discussion by ppb-rapid** [dedupeio/dedupe#1155] [Q&A] [answered]
I wish to automate the process of generating cluster Ids for my dataset without having to provide labeled pairs or any input other than the data and schema.

Is this possible with this library?

**Comment by ppb-rapid** [comment]
@fgregg @NickCrews Please help!

**Comment by NickCrews** [comment, marked as answer]
Dedupe does not support unsupervised. If you need unsupervised training then you will want to look at a different software package. This list is a bit outdated, but it's a reasonable starting point. https://github.com/J535D165/data-matching-software
I personally recommend splink.

**Comment by fgregg** [comment]
@NickCrews is right. i think there's some very interesting possibilities, but nothing in dedupe yet.

**Comment by ppb-rapid** [comment]
Thank you for the quick response and the list @NickCrews @fgregg ! Dedupe is wonderfully written, I learnt a lot.
